### PR TITLE
My portfolio project pull request

### PR DESCRIPTION
**Pull request by Kashif-Saleem-Ghuman**
## Portfolio template replica: milestone 1

In this milestone:

-Added index.html file for displaying the elements o the portfolio on the browser
-Added style.css file for styling the elements of the page. 
-Added an image folder that contains all the images used in the HTML file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <img id="logo" src="img/image_geometry_1.png" alt="">
+    <div id="bars">
+      <a class="active" href="#"><i class="fa fa-bars" style="color:black;"></i></a>
+    </div>
+  </nav>
+  <header class="heading">
+    <div id="name">
+      Kashif Saleem
+    </div>
+  </header>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ]  For Making your HTML markup is well structured and follows a clear hierarchy and semantic correct I recommend you to use `header` tag for Toolbar section with in `nav` tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  <intro class="intro">
+    <p id="para">
+      Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+      If you like what you see and have a project you need coded, don’t hesitate and contact me.
+    </p>
+  </intro>
+  <div class="button">
+    <a id="btn" href="">Start collaboraton</a>
+  </div>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] For Making your HTML markup is well structured and follows a clear hierarchy and semantic correct I recommend you to use `section` tag for headline section.
- [ ] I recommend you to use `h1` tag for main title("Hello! I am a software developer!") and `p` tag for paragraph("I can help you build a product, feature or website...")

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @Kashif-Saleem-Ghuman   , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Good README 
✔️ Good commit messages

almost

## Required Changes ♻️



_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Kashif-Saleem-Ghuman) [reply index.html]: **Please see if this is correct now**
Capture

Comment (Kashif-Saleem-Ghuman) [reply index.html]: **Please see if this is correct now**
changed

File: index.html (hunk 1 above)
Comment (Kashif-Saleem-Ghuman) [reply index.html]: Do you mean that I should replace the **nav** tag with a **header** tag?

**Comment by Kashif-Saleem-Ghuman** [comment]
I have replied to the comments with the changes made that you requested. Kindly check if those are correct. I shall then submit it for review.

Thank You.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

File: index.html (hunk 1 above)
Comment (Abenezer-Tilahun) [reply index.html]: Hello @Kashif-Saleem-Ghuman `nav` tag should be inside the `header` tag.

File: index.html (hunk 2 above)
Comment (Abenezer-Tilahun) [reply index.html]: Hello @Kashif-Saleem-Ghuman  good for using `section ` tag but still i recommend you to include `button`  inside the `section` tag and the design great.

**Comment by Kashif-Saleem-Ghuman** [comment]
Ok, I have done that and submitted it for review.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by Kashif-Saleem-Ghuman** [comment]
Oh but since you didn’t mention the button earlier, hence I did not include it and I have already submitted it for review.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

File: index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  </header>
+  <section class="intro">
+    <h1 id="intro-heading">
+      Hello! I am a software developer!
+    </h1>
+    <p id="para">
+      I can help you build a product, feature or website. Taka a look of my works.
+      If you like what you see and have a project you need coded, don’t hesitate and contact me.
+    </p>
+  </section>
```
Comment (leonardodiasb) [inline index.html]: - [ ] The header section is wrapping the nav menu in the top and the title of the page("Kashif Saleem") should be inside the first section of the page. Please move the title to the section.
 - [ ] The previous code reviewer asked you to use the `h1` tag, but in a not correct place. The "Hello! I am a software developer!" is part of the paragraph and it was correct when you first submitted the PR. The main title (e.g., "Kashif Saleem") should be an `h1` tag. If you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info. Kindly move your name inside the `h1` and move the "Hello..." to inside the `p` tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+</head>
+<body>
+  <header>
+    <nav class="container">
+      <img id="logo" src="img/image_geometry_1.png" alt="">
```
Comment (leonardodiasb) [inline index.html]: - [OPTIONAL] This is a background image (decorations are always background images, not `img` tags). If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.background image (decorations are always background images, not img tags).

If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Kashif-Saleem-Ghuman, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Good README
✔️ Good commit messages

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kashif-Saleem-Ghuman** [comment]
So the code reviewer misguided me ?, and because of that, I wasted a review. Will I get an extra review now without using up my extra reviews?

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

________________________________

**Review by Lameck1 — COMMENTED** [review]
Hello, @Kashif-Saleem-Ghuman 👋,

_This is an additional review as a result of the second opinion request._

The concerns of @leonardodiasb are genuine. He has just tried to give further direction on what @Abenezer-Tilahun really intended to point out. Please consider addressing the concerns @leonardodiasb has raised and ask for another code review if you have any review limit left. If you don't have any review limit left, work on the concerns then leave a message. I will take over from there.

Keep rocking 🚀!

**Comment by Kashif-Saleem-Ghuman** [comment]
Ok I have one review left so I will make changes and submit.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by Kashif-Saleem-Ghuman** [comment]
*   [ ] The header section is wrapping the nav menu in the top and the title of the page("Kashif Saleem") should be inside the first section of the page. Please move the title to the section.
I am unable to understand what exactly are you asking me to do here. Can u please explain to me?

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

File: index.html (hunk 3 above)
Comment (Kashif-Saleem-Ghuman) [reply index.html]: **Is this correct now?**
image
 **For the first suggestion are you implying that I change the tag of (KASHIF SALEEM) from header to section?**

File: index.html (hunk 4 above)
Comment (Kashif-Saleem-Ghuman) [reply index.html]: **is this correct now?**
image

**Comment by Kashif-Saleem-Ghuman** [comment]
***@***.***
I have made the changes. Kindly check if it is ok now?

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

________________________________

File: index.html [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  </section>
+  <section class="intro">
+    <p id="intro-heading">
+      Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+      If you like what you see and have a project you need coded, don’t hesitate and contact me.
+    </p>
+  </section>
+  <div class="button">
+    <a id="btn" href="">Start collaboraton</a>
+  </div>
```
Comment (Lameck1) [inline index.html]: You are almost there @Kashif-Saleem-Ghuman. Let's make the whole of this to be one section. Your name, the paragraph, and the button should all be part of one section. We often call this first section the `hero section`. It is the landing page of your portfolio.

Please refactor your code as below and we'll be good to go:

```JavaScript
<section class="heading">
  <h1 id="name">Kashif Saleem</h1>
  <p id="intro-heading">
    Hello! I am a software developer! I can help you build a product, feature, or website. Taka a look at my work.
    Don't hesitate to contact me if you like what you see and have a project you need to be coded.
  </p>
  
  <div class="button">
    <a id="btn" href="">Start collaboraton</a>
  </div>
</section>
```

**Review by Lameck1 — COMMENTED** [review]
Hello @Kashif-Saleem-Ghuman,

This is a manual review.

Based on the very last review you received from @leonardodiasb, for you to address the concerns from the review, you need to consider the code example discussed in the inline comment below. Please do this last change and we'll go ahead and approve the project once you are done.

Cheers ✌🏼and happy coding.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @Kashif-Saleem-Ghuman  
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Abenezer-Tilahun** [comment]
Hello @Kashif-Saleem-Ghuman  I am really sorry for taking your time for misguided  you for pointing out use of `h1` tag for incorrect place.

**Comment by Kashif-Saleem-Ghuman** [comment]
Abenezer I appreciate your apology. Apologies accepted. It did get me into a bit of panic but all is well that ends well.
Thank you.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Pull request merged**